symphony/bdk/core/config/model/bdk_ssl_config.py

```diff
@@ -0,0 +1,10 @@
+from symphony.bdk.core.config.model.bdk_certificate_config import BdkCertificateConfig
+
+
+class BdkSslConfig:
+
+    def __init__(self, config):
+        if config is not None:
+            self.trust_store = BdkCertificateConfig(**config.get("trustStore")) if "trustStore" in config else BdkCertificateConfig()
```

Comment:
Please put a trustStore param in the constructor, we don't need the full config dict.